docs/modules/usage/cloud/cloud-forgejo-resolver.md

```diff
@@ -0,0 +1,133 @@
+# Forgejo Integration
```

Comment:
This doesn't belong in `cloud`--that's for app.all-hands.dev where we likely won't support this near term

Reply:
openhands tasked to resolve this